chapter3.adoc

```diff
@@ -6,7 +6,13 @@
 under which a hart is operating. The interface consists of M-mode CSRs `msdcfg`
 and `mttp`. The SDID programmed via this interface is a local identifier for the
 hart and may be used to tag hart-local resources to access-control data
-associated with the supervisor domain.
+associated with the supervisor domain. For most typical usages, the SDID will
+remain stable over the life of the supervisor domain. However, the SDID can be
+changed at the cost of fencing to flush any prior cached state. The SDID is
+unique only to a hart, hence, SDID must not be used to tag downstream
+transactions that are globally visible - implementations may map the SDID from
+the initiator to other (global) identifiers but that mapping is
+implementation-specific and out of scope of this specification.
```

Comment:
If statement 2 from #76 is correct, then we shouldn't suggest mapping the SDID to anything outside the hart:

```suggestion
transactions that are globally visible.
```